README.md

```diff
@@ -97,6 +97,12 @@ app.use(checkUser);
 app.use(subdomain('v1.api', router));
 ```
 
+If you want to handle any subdomain you can use the following.
```

Comment:
@juliocanares thanks for the PR - much appreciated! I think we should rename `Gotchas` below to `FAQs` and that's where this should live.